packages/material-ui/src/AccordionDetails/AccordionDetails.js

```diff
@@ -1,19 +1,54 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
-import withStyles from '../styles/withStyles';
+import { deepmerge } from '@material-ui/utils';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
+import { getAccordionDetailsUtilityClass } from './accordionDetailsClasses';
 
-export const styles = (theme) => ({
-  /* Styles applied to the root element. */
-  root: {
-    padding: theme.spacing(1, 2, 2),
+const overridesResolver = (props, styles) => {
```

Comment:
Can be simplified to:

		 const overridesResolver = (props, styles) => styles.props || {};